test/sql/function/time/test_date_part.test

```diff
@@ -95,3 +95,6 @@ SELECT dayofyear( FROM i) FROM times
 
 statement error
 SELECT week( FROM i) FROM times
+
+statement error
+SELECT yearweek( FROM i) FROM times
```

Comment:
I didn't notice this before, but this seems like it gives a syntax error rather than actually testing the behavior you want to test. I assume you meant `SELECT EXTRACT(yearweek FROM i) FROM times` here?

Reply:
Doh! Copy and paste error. I'll fix that now.

Reply:
Sadly there doesn't seem to be a way to check the actual message?

Reply:
The sqllogictest api only allows you to check whether or not an error has occurred. We do have some [blacklisted exceptions here](https://github.com/cwida/duckdb/blob/master/test/sqlite/test_sqllogictest.cpp#L628) that cause a failure even on a `statement error`. I think adding a parser exception there would make sense. We could also add some additional machinery to actually parse the error, but I would like to avoid dependencies on specific error messages that cause test failures if the error messages change.

Reply:
That sounds like a good trade off. In particular, I just copied and pasted the Postgres errors from a shell(!) and at some point we might want different ones that are more like SQLite?